shoes-core/lib/shoes/builtin_methods.rb

```diff
@@ -37,6 +37,13 @@ def error(message = '')
       Shoes.logger.error message
     end
 
+    def exit(message = '')
+      Shoes::LOG << ['exit', message]
```

Comment:
I'm not sure that we actually need to have a `message` passed with exit. Did you pick that up from the other built-in methods nearby, or some other spot?